source/sections/_lose_the_spreadsheet.html.erb

```diff
@@ -21,6 +21,9 @@
             <%= image_tag "icons/#{col.icon}.svg", class: "icon" %>
             <h3 class="heading"><%= col.title %></h3>
             <p><%= col.description %></p>
+            <a href="<%= col.link %>" title="Learn More" class="featured_link__button btn--secondary btn btn-solid">
```

Comment:
As it's not on _staging_ yet, I can't check it out, but from your screenshot in the description, this might want a `margin-top` on it.

Or perhaps just a linebreak before:
```suggestion
            <br />
            <a href="<%= col.link %>" title="Learn More" class="featured_link__button btn--secondary btn btn-solid">
```